DevMVCComponent/DevMVCComponent/Processor/CacheProcessor.cs

```diff
@@ -71,8 +71,8 @@ public class CacheProcessor {
         /// <param name="name"></param>
         /// <returns></returns>
         public object Get(string name) {
-            if (HttpContext.Current.Cache != null && HttpContext.Current.Cache[name] != null) {
-                return HttpContext.Current.Cache[name];
+            if (HttpRuntime.Cache != null && HttpRuntime.Cache[name] != null) {
```

Comment:
Thank you @nazmi720 for taking your valuable time , providing a fix. Can we do this

GetCacheObject() -> Return either `HttpContext.Current.Cache` or  ` HttpRuntime.Cache`

Depending on which one exist then both would work. 
Thank you once again.

Reply:
I guess that might become problematic.

Like, if we are requesting from the main thread, then we will get one cache object, and requesting from a background thread, we would get a separate cache. That might be problematic in some cases.

I guess you would review my suggestions and take the best decision.

Thank you.

Reply:
ok then taking your solution